specification.md

```diff
@@ -1950,7 +1950,7 @@ Once all actions have been performed, a transition to another state can occur.
 | [stateDataFilter](#state-data-filter) | State data filter | object | no |
 | [onErrors](#Error-Definition) | States error handling and retries definitions | array | no |
 | eventTimeout | If eventConditions is used, defines the time period to wait for events (ISO 8601 format). For example: "PT15M" (15 minutes), or "P2DT3H4M" (2 days, 3 hours and 4 minutes)| string | yes only if eventConditions is defined |
-| default | Default transition of the workflow if there is no matching data conditions or event timeout is reached. Can be a transition or end definition | object | yes |
+| default | Default transition of the workflow if there is no matching data conditions or event timeout is reached. Can be a transition or end definition | object | no |
```

Comment:
Imho, if default is not provided, then we should treat it like the user specified an 'end' definition, and that the doc should be updated here to clearly call that out.  On the call, the alternative idea was to say that a runtime exception should be thrown, imho these are functionally the same but 'end' seems less surprising.

Reply:
imho exception is better as modellers can catch it during testing. typically not being able to transition as wanted from a switch state is a really bad thing and masking it with an "end" (workflow termination) might be very confusing imo. wdyt?